the cdc server panics when stopping a changefeed

## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

When using the API to stop a change feed, sometimes, it causes the server panic.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1a148f2]

goroutine 248 [running]:
github.com/pingcap/ticdc/cdc.(*ddlHandler).receiveDDL(0xc000b97450, 0x0, 0x0, 0x0)
        github.com/pingcap/ticdc@/cdc/owner_operator.go:79 +0x42
github.com/pingcap/ticdc/cdc.newDDLHandler.func2(0xc000a4a768, 0x0)
        github.com/pingcap/ticdc@/cdc/owner_operator.go:67 +0x11c
golang.org/x/sync/errgroup.(*Group).Go.func1(0xc000b897d0, 0xc000b89950)
        golang.org/x/sync@v0.0.0-20190911185100-cd5d95a43a6e/errgroup/errgroup.go:57 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
        golang.org/x/sync@v0.0.0-20190911185100-cd5d95a43a6e/errgroup/errgroup.go:54 +0x66
```

This is found in the integration test #431 

2. What did you expect to see?

The replication job should be stopped.

3. What did you see instead?

The server process crashed

4. Versions of the cluster

 - master

--- a/cdc/owner_operator.go
+++ b/cdc/owner_operator.go
@@ -64,6 +64,9 @@ func newDDLHandler(pdCli pd.Client, checkpointTS uint64) *ddlHandler {
 			case <-ctx.Done():
 				return ctx.Err()
 			case e := <-rawDDLCh:
+				if e == nil {
+					continue
+				}
 				err := h.receiveDDL(e)
 				if err != nil {
 					return errors.Trace(err)

